Library.Business/DTO/Person/PersonDto.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Library.Business.DTO.Person
+{
+    public class PersonDto
+    {
+        public long Id { get; set; }
+
+        public string FirstName { get; set; }
+#nullable enable
```

Comment:
perhaps you should avoid directives (#nullable) and try to move such configuration to project settings if possible

Reply:
If I apply nullable for all project, every String, List will result a warning, which says I have to set this variables as nullable "?", but i dont want to do some strings nullable. 
Looks like this has to be still investigated.